### DISCUSSION TITLE
Windows App SDK 0.8.1 is now available!

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#1056] [Announcements]
### Bugs fixed in Windows App SDK 0.8.1:
- Windows App SDK cannot run on latest Insider build
- Crash in editable ComboBox when entering a value that does not appear in dropdown
- WebView2 doesn't allow user to tab out once focused has been received
- Fully qualify Windows.Foundation.Metadata.DefaultOverload namespace in WinUI generated code to avoid namespace ambiguity 
    - This fixes bug [#5108](https://github.com/microsoft/microsoft-ui-xaml/issues/5108).
- Security fix – see [CVE-2021-34489](https://msrc.microsoft.com/update-guide/en-US/vulnerability/CVE-2021-34489) for more details.

The limitations and known issues for v0.8 also apply to v0.8.1, unless marked otherwise in [our documentation](https://docs.microsoft.com/windows/apps/windows-app-sdk/stable-channel#limitations).

**Comment by shelllet** [comment]
small updates

**Comment by pjmagee** [comment]
> ## Windows App SDK 0.8.1 is a stable servicing release that fixes a few critical bugs.
> ### Bugs fixed in Windows App SDK 0.8.1:
> * Windows App SDK cannot run on latest Insider build
> * Crash in editable ComboBox when entering a value that does not appear in dropdown
> * WebView2 doesn't allow user to tab out once focused has been received
> * Fully qualify Windows.Foundation.Metadata.DefaultOverload namespace in WinUI generated code to avoid namespace ambiguity
>   
>   * This fixes bug [#5108](https://github.com/microsoft/microsoft-ui-xaml/issues/5108).
> * Security fix – see [CVE-2021-34489](https://msrc.microsoft.com/update-guide/en-US/vulnerability/CVE-2021-34489) for more details.
> 
> The limitations and known issues for v0.8 also apply to v0.8.1, unless marked otherwise in [our documentation](https://docs.microsoft.com/windows/apps/windows-app-sdk/stable-channel#limitations).

Do these fixes also get applied to the experimental channel of 0.8?
